resources/route53-resolver-rules.go

```diff
@@ -0,0 +1,114 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53resolver"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+type (
+	// Route53ResolverRule is the resource type
+	Route53ResolverRule struct {
+		svc        *route53resolver.Route53Resolver
+		id         *string
+		name       *string
+		domainName *string
+		vpcIds     []*string
+	}
+
+	association struct {
+		id    *string
+		vpcID *string
+	}
```

Comment:
AFAICS this struct isn't used anywhere, is it?

Reply:
Nope, good catch.